src/components/SummaryBar/SummaryBar.jsx

```diff
@@ -88,7 +89,7 @@ const SummaryBar = props => {
       console.log('User Tasks not loaded.');
```

Comment:
Hi Ram, 
Do we need to have this uncommented? If not then better to comment it out.

Reply:
Hi  Parth-tech ,
I am also not sure about that changes, because I haven't make those changes and not related to this changeset